test/nogc.d

```diff
@@ -0,0 +1,5 @@
+extern(C) __gshared string[] rt_options = [ "gcopt=gc:conservative" ];
```

Comment:
The idea was to select a non-existing gc `"gcopt=gc:non-existing"`, so to test that the GC is indeed never initialized.